src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ArrivalsServiceConfirmArrivalTest.kt

```diff
@@ -43,50 +44,107 @@ class ArrivalsServiceConfirmArrivalTest {
 
   @Test
   fun `Confirm arrival throw exception when location is empty`() {
-    whenever(basmService.getArrival(any())).thenReturn(ARRIVAL_PROTOTYPE.copy(prisonNumber = "ABC123A"))
+    whenever(basmService.getArrival(any())).thenReturn(ARRIVAL_PROTOTYPE.copy(prisonNumber = PRISON_NUMBER))
+    whenever(prisonerSearchService.getCandidateMatches(any())).thenReturn(
+      listOf(
+        MatchPrisonerResponse(
+          firstName = FIRST_NAME,
+          lastName = LAST_NAME,
+          dateOfBirth = DATE_OF_BIRTH,
+          prisonerNumber = PRISON_NUMBER,
+          pncNumber = PNC_NUMBER,
+          status = "INACTIVE OUT",
+        )
+      )
+    )
     whenever(prisonService.admitOffenderOnNewBooking(any(), any())).thenReturn(INMATE_DETAIL_NO_UNIT)
     assertThatThrownBy {
       arrivalsService.confirmArrival(MOVE_ID, CONFIRMED_ARRIVAL_DETAIL_PROTOTYPE)
-    }.hasMessage("Prisoner: 'ABC123A' do not have assigned living unit")
+    }.hasMessage("Prisoner: '$PRISON_NUMBER' do not have assigned living unit")
   }
```

Comment:
this test wasn't valid as it was relying on prison number provided from BASM when there was no match